### PR TITLE
markup change to Boehm-GC

### PR DESCRIPTION
**Pull request by xsuchy**
To accomodate the license found in mariadb
Fixes: #2507

### CODE REVIEW

File: src/Boehm-GC.xml [hunk 1]
```diff
@@ -14,8 +15,8 @@
             WARRANTY EXPRESSED OR IMPLIED. ANY USE IS AT YOUR OWN RISK.
          </p>
          <p>
-            Permission is hereby granted to use or copy this program for any
-            purpose, provided the above notices are retained on all copies.
+            Permission <optional>is hereby granted to</optional> use or copy this <alt name="program" match="program|software">program</alt> for any
```
Comment (richardfontana) [inline src/Boehm-GC.xml:18]: I think this has to be 
```
<optional>is hereby granted</optional> to use [etc.]

Comment (xsuchy) [reply src/Boehm-GC.xml]: True. Updated.

File: src/Boehm-GC.xml [hunk 2]
```diff
@@ -14,8 +15,8 @@
             WARRANTY EXPRESSED OR IMPLIED. ANY USE IS AT YOUR OWN RISK.
          </p>
          <p>
-            Permission is hereby granted to use or copy this program for any
-            purpose, provided the above notices are retained on all copies.
+            Permission <optional>is hereby granted to</optional> use or copy this <alt name="program" match="program|software">program</alt> for any
+            purpose, <optional>is hereby granted without fee,</optional> provided the above notices are retained on all copies.
             Permission to modify the code and to distribute modified code is
```
[the remark this answers is not in the document]
Comment (swinslow) [reply src/Boehm-GC.xml:20]: @richardfontana I'd say probably not (i.e. it probably doesn't matter).

Agreed that this is in theory an issue for lots of SPDX XML files, and especially those with "anything goes" / `.*` regular expressions. However, I think in practice we accept that this is the practice at present. I'm not aware of any "real world" license texts that have tried to abuse it in a meaningful way.

File: src/Boehm-GC.xml [hunk 3]
```diff
@@ -14,8 +15,8 @@
             WARRANTY EXPRESSED OR IMPLIED. ANY USE IS AT YOUR OWN RISK.
          </p>
          <p>
-            Permission is hereby granted to use or copy this program for any
-            purpose, provided the above notices are retained on all copies.
+            Permission <optional>is hereby granted</optional> to use or copy this <alt name="program" match="program|software">program</alt> for any
+            purpose, <optional>is hereby granted without fee,</optional> provided the above notices are retained on all copies.
```
Comment (swinslow) [inline src/Boehm-GC.xml:19]: I'm not sure this actually matches https://github.com/MariaDB/server/blob/11.6/libmysqld/lib_sql.cc with the present edits. Note the comma after "purpose" in Boehm-GC which isn't present in the MariaDB link.

Might be able to make that optional, but ugh... to @richardfontana's point, this does start to look a bit ugly...

**Comment by xsuchy** [comment]
Closing per discussion in the associated issue.

**Pull request closed without merging**